### ISSUE TITLE
`no-multi-spaces` does not support ES7 class properties

### ISSUE THREAD

**Issue by alexilyaev** [babel/eslint-plugin-babel#89]
In my team we like to align variable declarations, so we use `no-multi-spaces` to enforce that.
But we recently started working with `babel-eslint` to use ES7 features  and our lint fails when we try to align class properties.

Code:

``` js
export class Login extends React.Component {
  canSubmit   = false;
  serverError = null;
}
```

.eslintrc:

``` json
{
  "plugins": [
    "react",
    "babel"
  ],

  "extends": ["eslint:recommended", "plugin:react/recommended"],

  "env": {
    "es6": true,
    "browser": true
  },

  "parser": "babel-eslint",
  "parserOptions": {
    "ecmaVersion": 6,
    "ecmaFeatures": {
      "jsx": true
    },
    "sourceType": "module"
  },

  "rules": {
    "no-multi-spaces": [1, {
      "exceptions": {
        "VariableDeclarator": true,
        "ImportDeclaration": true,
        "ImportSpecifier": true,
        "AssignmentExpression": true
      }
    }]
  }
}
```

Warning:

```
warning  Multiple spaces found before '='        no-multi-spaces
```

**Comment by OscarBarrett** [comment]
You can work around this by adding `ClassProperty` to the list of exceptions.

**Comment by alexilyaev** [comment]
Oops, found that a few months ago, forgot to mention it here, closing.
